src/main/java/com/jcabi/github/Comment.java

```diff
@@ -86,6 +88,18 @@
      */
     void remove() throws IOException;
 
+    /**
+     * Adds the reaction to the comment.
+     * @param reaction Reaction to be added.
+     */
+    void react(Reaction reaction);
+
+    /**
+     * List the reactions of the comment.
+     * @return Comment reactions.
+     */
+    Collection<Reaction> reactions();
```

Comment:
@paulodamaso I think we should have the ``Reactions extends Iterable<Reaction>`` interface, what do you think? 
Object of type ``Reactions`` would also offer methods for removing a reaction, for instance.

What do you think?

Reply:
@amihaiemil I totally agree with both comments, I'll fix it very soon

Reply:
@paulodamaso Right, let's leave it as it is for now. But nore that my proposed ``Reactions`` wouldn't be used as you described. It would be something like:

```java
final Reactions reacts = issue.comments.get(...).reactions();//reactions of a Comment.
reacts.add(...);
reacts.remove(reactionId);
for(Reaction react : reacts){
    //iterate all reactions of that comment.
}
```

But, as I said, let's leave it like this for now.